.github/workflows/publish-packages.yml

```diff
@@ -0,0 +1,62 @@
+name: Publish Packages
+run-name: Publish Packages ${{ inputs.only-distro }} ${{ inputs.only-codename }} ${{ inputs.only-component }}
+
+on:
+  workflow_dispatch:
```

Comment:
question: can you describe at a high level how you envision this workflow to be called?

Reply:
This is predominantly being triggered as a re-usable workflows inside `build-deb-v4.yml` actions. I specifically pulled it out to its own standalone action, that it can be triggered manually if there was ever a scenario where `build-deb-v4.yml` fails, for example, a package source has a conflict, or the `build-deb-v4.yml` was cancelled, or any other reasons (albeit very slim chance that there ever is a reason).

In that case we have the possibility to fix the issue (e.g. going on publish server and cleanup something, fix other thing, etc) and just triggering this workflow.

But in all reality, I highly doubt we'll ever run this independently. It's just a bit cleaner in the code and makes the, already too big, `build-deb-v4.yml` a little bit smaller and more readable.